app/views/layout.erb

```diff
@@ -51,7 +57,7 @@
       ## a  a  .
       (   '._)
        |'-- |
-     _.\___/_   ___<label><input type="checkbox" name="pjax" />pjax</label>___
+     _.\___/_   ___<label><input type="checkbox" name="pjax" checked />pjax</label>___
```

Comment:
:+1: for making the `checked` state by default for this demo, but I would prefer if changes *unrelated* to the form submit pjaxing (the primary topic of your PR) were made as separate commits. It's good git practice in general for any project.